src/test/java/io/lettuce/core/cluster/AdvancedClusterClientIntegrationTests.java

```diff
@@ -339,6 +339,20 @@ void flushall() {
         assertThat(dbsize).isEqualTo(0);
     }
 
+    @Test
+    void flushallAsync() throws InterruptedException {
+
+        writeKeysToTwoNodes();
+
+        assertThat(sync.flushallAsync()).isEqualTo("OK");
+
+        // This is hacky, but by its nature FLUSHALL ASYNC doesn't give us a mechanism to know when it's done
+        Thread.sleep(1000);
```

Comment:
Let's switch this test to issue `GET` commands to keys written in `writeKeysToTwoNodes`. Using `GET` we no longer introduce a dependency to timing and our tests run in a shorter time.

Reply:
Done and pushed.